docs/developer-docs/getting-started/deploy-and-manage.mdx

```diff
@@ -0,0 +1,335 @@
+---
+keywords: [beginner, getting started, tutorial, deploy canisters, deploy smart contract, deploy project, deploy, deploy locally, deploy mainnet, manage smart contract, manage project, manage canister, set freezing threshold, check cycles balance, top up canister, start canister, stop canister, canister settings]
+---
+
+import TabItem from "@theme/TabItem";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import { AdornedTabs } from "/src/components/Tabs/AdornedTabs";
+import { AdornedTab } from "/src/components/Tabs/AdornedTab";
+import { BetaChip } from "/src/components/Chip/BetaChip";
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import { GlossaryTooltip } from "/src/components/Tooltip/GlossaryTooltip";
+
+# Deploy and manage
+
+<MarkdownChipRow labels={["Beginner", "Getting started", "Tutorial"]} />
+
+## Overview
+
+Once a canister's code has been written, it must be compiled into Wasm, installed into the canister itself, and then deployed to a network. The `dfx deploy` command executes each of these tasks as part of the deployment process; however, `dfx` commands also exist for running each step individually.
+
+It is recommended to deploy canisters locally or to the temporary [playground](/docs/current/developer-docs/smart-contracts/deploy/overview#deploying-to-playground) network to test canister functionality and make adjustments before considering the canister production-ready and deploying it to the mainnet. Deploying canisters locally and to the playground is free and does not cost cycles. Once a canister is deployed to the mainnet, it will consistently consume cycles and will need to be topped up periodically.
```

Comment:
Should we make it more clear on what is available on local vs. mainnet? We get a lot of questions and comments on the forum about certain features working fine locally (mostly consensus-required features such as HTTPS Outcalls) but not on mainnet because these features do not go through consensus locally.

Reply:
depending on how much the feature support differs I think that might deserve a dedicated page which we could refer here 🤔 

do we have a list for that somewhere @jennifertrin ?

Reply:
We can expand the [Deploy page](https://internetcomputer.org/docs/current/developer-docs/smart-contracts/deploy/overview) to include additional details about these differences.

Reply:
I think we can expand the Deploy page - I think we can also consider it for later. It doesn't need be in this PR.